.github/pull_request_template.md

```diff
@@ -0,0 +1,38 @@
+## Description
+
+> Provide a brief description about **why** this PR is necessary. Be sure to provide context.
+
+## Related issues and/or PRs
+
+> If this PR addresses or references any issues and/or other PRs, list them here.
+
+## Changes made
+
+> Outline the specific changes made in this pull request. Include relevant details, such as added features, bug fixes, code refactoring, or improvements.
+
+## How these changes were tested
+
+> Describe the testing that you have performed to ensure the changes are working as expected. Include details about the test cases or scenarios covered.
+
+## Checklist
+
+> The following is a best-effort checklist. If any items in this checklist are not applicable to this PR or are dependent on other, unmerged PRs, please still mark the checkboxes after you have read and understood each item.
+
+- [ ] I have commented my code, particularly in hard-to-understand areas.
+- [ ] I have updated the documentation to reflect the changes.
+- [ ] Any remaining open issues linked to this PR are documented and up-to-date (Jira, GitHub, etc.).
+- [ ] Tests (unit, integration, etc.) have been added for the changes.
+- [ ] My changes generate no new warnings.
+- [ ] Any dependent changes in other PRs have been merged and published.
+
+## Additional notes (optional)
+
+> Provide any additional information or notes that may be relevant to the reviewers or stakeholders.
+
+## Release notes
+
+> Provide release note text for this PR based on the guidelines at [Guidelines for Writing Release Notes](https://developers.scalar-labs.com/docs/style-guide/release-notes/).
+> 
+> For example:
+> 
+> Added validation for the Consensus Commit mutation operation.
```

Comment:
```suggestion
```

@ypeckstadt @thongdk8 
I think we don't have any release notes for the Helm Chart of ScalarFlow. Is my understanding correct?
If yes, I think we can remove this section at this time.
However, if you have any plans to create release notes in the future, I think it's OK to remain this section.

What do you think?

Reply:
Yes, currently we dont have but we will add it anyways, so it is fine to keep the above section I think.

Reply:
Yeah, I think it is better to keep it as in the end ScalarFlow helm charts will be public. So there will be release notes.